creator-node/src/routes/files.js

```diff
@@ -192,13 +192,34 @@ module.exports = function (app) {
       if (req.query.filename) {
         res.setHeader('Content-Disposition', contentDisposition(req.query.filename))
       }
+
+      // Determine if cat one byte of ipfs CID takes over 500ms
```

Comment:
it's not clear from this code that `ipfsSinglebyteCat()` will throw if cat exceeds 500ms. a comment to that effect here would be helpful